src/masternodes/mn_checks.cpp

```diff
@@ -2922,15 +2924,15 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
                 }
                 else
                 {
-                    subInDFI = kv.second;
+                    subInDFI = tokenLoanBalance;
                 }
 
                 CDataStructureV0 liveKey{AttributeTypes::Live, ParamIDs::Economy, EconomyKeys::PaybackDFITokens};
                 auto balances = attributes->GetValue(liveKey, CBalances{});
-                auto penaltyDFI = MultiplyAmounts(subInDFI, COIN - penaltyPct);
+                auto tokenEquivalentDFI = MultiplyAmounts(subInDFI, COIN - penaltyPct);
```

Comment:
but this is not the tokenEquivalent. This is only the penalty (the 1% currently). or am I mistaken here? (penaltyPct = 99% currently)

Reply:
Please ignore. Superceded by 1089